mne/viz/misc.py

```diff
@@ -426,16 +445,16 @@ def _plot_mri_contours(mri_fname, surfaces, src, orientation='coronal',
             if ai < n_col or n_col == 1:  # top
                 ax.text(dat.shape[1] / 2., dat.shape[0] - 1, ylabels[1],
                         ha='center', va='top', **kwargs)
-        if img_output:
-            output = BytesIO()
-            fig.savefig(output, bbox_inches='tight',
-                        pad_inches=0, format='png', dpi=dpi)
-            out.append(base64.b64encode(output.getvalue()).decode('ascii'))
```

Comment:
I like the idea of returning a list of ndarrays. As you mention correctly, the downstream plotting functions should handle this just fine, unless there are bugs somewhere (which obviously would need to be fixed anyway)

I will look into this after ensuring that ndarray plotting works in general – you reported some issue there with a deprecated function, which I'll look into just now.

Reply:
You shouldn't need to do that, I pushed removal of the previously added (in this PR) `tight_layout` call. So ndarrays should work now

Reply:
(I added an edit to say by "that" I meant the ndarray fixes, I pushed a commit for that already -- but the changes to make this actually return ndarrays instead of a list of figures would still be good)

Reply:
got you, thanks!